content/source/docs/enterprise/saml/configuration.html.md

```diff
@@ -11,19 +11,47 @@ SAML requires the configuration of two parties:
-2. **Single Log Out URL**:  specifies the HTTP(s) endpoint on your IdP for single logout requests. This value is provided by your IdP configuration. Single Logout is not yet supported.
-3. **Identity Provider Certificate**: Specifies the PEM encoded X.509 Certificate as provided by the IdP configuration.
-4. **Team Attribute Name**: (default: `MemberOf`) Specifies the name of the SAML attribute that determines [team membership](./team-membership.html). The value of this attribute in the SAML assertion must be a string containing a comma-separated list of team names.
-5. **Site Admin Role**: (default: `site-admins`) Specifies the role for site admin access, provided in the list of roles sent in the Team Attribute Name attribute.
+[admin]: ../private/admin/index.html
+
+Go to the SAML section of the site admin pages. You can use the "Site Admin" link in the upper-right user icon menu, or go directly to `https://<TFE HOSTNAME>/app/admin/saml`.
+
+Once there, enter values (or leave the default values) for TFE's SAML settings, then click the "Save SAML Settings" button at the bottom of the page.
+
+The SAML settings are separated into sections:
+
+### SAML Settings
+
+- **Enable SAML single sign-on**: This checkbox must be enabled.
+
+### Identity Provider Settings
+
+- **Single Sign-On URL**: The HTTP(S) endpoint on your IdP for single sign-on requests. This value is provided by your IdP configuration.
+- **Single Log-Out URL**: The HTTP(s) endpoint on your IdP for single logout requests. This value is provided by your IdP configuration. Single Logout is not yet supported.
+- **IdP Certificate**: The PEM encoded X.509 Certificate as provided by the IdP configuration.
+
+### Attributes
+
+- **Username Attribute Name**: (default: `Username`) The name of the SAML attribute that determines the TFE username for a user logging in via SSO.
+- **Site Admin Attribute Name**: (default: `SiteAdmin`) The name of the SAML attribute that determines whether a user has site-admin permissions. The value of this attribute in the SAML assertion must be a boolean. Site admins can manage settings and resources for the entire PTFE instance; see [Administering Private Terraform Enterprise][admin] for details.
+- **Team Attribute Name**: (default: `MemberOf`) The name of the SAML attribute that determines [team membership](./team-membership.html). The value of this attribute in the SAML assertion must be a string containing a comma-separated list of team names.
+
+### Team Membership Mapping
+
+- **Site Admin Role**: (default: `site-admins`; make blank to disable) An alternate way of managing site-admin permissions; if a role with this name is present in the value of the Team Attribute Name attribute, the user is an admin.
+
+    We recommend using the "site admin attribute name" setting instead. If you are using the site admin attribute, disable "site admin role" by deleting its value.
```

Comment:
People don't really need to delete this (it's overridden by the other option, so can be a good backup), so might be better to say "you can".

Reply:
Oh, I thought we were encouraging disabling it to avoid mistakes or whatever. But I guess it's probably not an easy mistake to make.